R/0_documentation.R

```diff
@@ -79,56 +79,78 @@ rd_target_schema <- function(type = "param") {
 
 rd_training_length <- function(type = "param") {
   checkmate::assert_choice(type, c("param", "field"))
-  paste("training_length (`numeric`)\\cr",
+  paste("(`numeric`)\\cr",
         "The number of days that should be included in the training of the model.",
         ifelse(type == "field", " Read only.", ""))
 }
 
 
 rd_start_date <- function(type = "param") {
   checkmate::assert_choice(type, c("param", "field"))
-  paste("start_date (`Date`)\\cr",
+  paste("(`Date`)\\cr",
         "Study period start.",
         ifelse(type == "field", " Read only.", ""))
 }
 
 
-rd_end_date <- function(type = "param") {
+rd_slice_ts <- function(type = "param") {
   checkmate::assert_choice(type, c("param", "field"))
-  paste("end_date (`Date`)\\cr",
-        "Study period end.",
+  paste("(`Date` or `character`)\\cr",
+        "Date to slice the database on (used if source_conn is a database).",
```

Comment:
The function says "slice_ts", so I assume it's analogous to `SCDB::slice_ts`, which supports timestamps:

```suggestion
        "Date or timestamp (parseable by `as.POSIXct`) to slice the database on (used if source_conn is a database).",
```